client/packages/common/src/ui/components/inputs/Autocomplete/utils.tsx

```diff
@@ -23,9 +24,11 @@ export const defaultOptionMapper = <T extends { [K in keyof T]: T[K] }>(
 
 export const getDefaultOptionRenderer: <T>(
   key: keyof T
-) => AutocompleteOptionRenderer<T> = key => (props, item) =>
-  (
-    <DefaultAutocompleteItemOption {...props}>
-      <Typography>{String(item?.[key])}</Typography>
+) => AutocompleteOptionRenderer<T> = labelKey => (props, item) => {
+  const key = (item as unknown as RecordWithId).id;
```

Comment:
can't remember why - doesn't hurt to have it in tho

Reply:
`(item as unknown as RecordWithId).id` -- interesting ;)